include/IECoreGL/Primitive.h

```diff
@@ -96,12 +96,12 @@ class IECOREGL_API Primitive : public Renderable
 		/// Most classes will not need to override this method - reasons for overriding would be
 		/// to substitute in custom geometry or vertex shaders and/or to bind in attributes
 		/// not already specified with addUniformAttribute() or addVertexAttribute().
-		virtual const Shader::Setup *shaderSetup( const Shader *shader, State *state ) const;
+		virtual const Shader::Setup *shaderSetup( const Shader *shader, State *state, bool vertexCsActive = true ) const;
```

Comment:
Changing the signature of this virtual method will break binary compatibility. Also, it will prevent the CurvesPrimitive and PointsPrimitive overrides from being picked up (because the signatures no longer match).